Classes/PBCommitList.m

```diff
@@ -43,7 +43,10 @@ - (void)keyDown:(NSEvent *)event
 	}
 	else if ([character rangeOfCharacterFromSet:[NSCharacterSet characterSetWithCharactersInString:@"jkcv"]].location == 0)
 		[webController sendKey: character];
-	else
+	else if (([character characterAtIndex:0] == NSDownArrowFunctionKey)
+			 && [event modifierFlags] & NSControlKeyMask) {
+		[controller selectParentCommit:self];
+	} else
```

Comment:
Actually, I think I'd prefer not adding a menu item just for that, and use ⌥↑/↓ (the rationale being that it's an "alternate" move up/down thingy. Right now that shortcut does Begin/End, which I personally don't find useful.

And I didn't do move-up because it was harder, and I think handling that would require a bunch of children-caching when revwalking.